sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -4,7 +4,7 @@
 # license information.
 # -------------------------------------------------------------------------
 import binascii
-from typing import Optional, Any, Union, Mapping
+from typing import Any, Dict, List, Mapping, Optional, Union
 from requests.structures import CaseInsensitiveDict
```

Comment:
same here about case insensitive dict...should use the one from azure core

Reply:
Yes this needs to be removed

Reply:
Same as async, has updated.